### ISSUE TITLE
Issues with release 1.49 reported by CPAN-testers

### ISSUE THREAD

**Issue by jonasbn** [jonasbn/perl-workflow#52]
Example CPAN-testers report demonstrating the issue

REF: http://www.cpantesters.org/cpan/report/0448edfe-560b-11eb-846b-3e0ef3237c92

```
From: David Cantrell (DCANTRELL)
Subject: FAIL Workflow-1.49 5.30.2 GNU/Linux
Date: 2021-01-14T01:51:28Z

This distribution has been tested as part of the CPAN Testers
project, supporting the Perl programming language.  See
http://wiki.cpantesters.org/ for more information or email
questions to cpan-testers-discuss@perl.org


--
Dear jonasbn,

This is a computer-generated report for Workflow-1.49
on perl 5.30.2, created by CPAN-Reporter-1.2018.

Thank you for uploading your work to CPAN.  However, there was a problem
testing your distribution.

If you think this report is invalid, please consult the CPAN Testers Wiki
for suggestions on how to avoid getting FAIL reports for missing library
or binary dependencies, unsupported operating systems, and so on:

http://wiki.cpantesters.org/wiki/CPANAuthorNotes

Sections of this report:

    * Tester comments
    * Program output
    * Prerequisites
    * Environment and other context

------------------------------
TESTER COMMENTS
------------------------------

Additional comments from tester:

this report is from an automated smoke testing program
and was not reviewed by a human for accuracy

------------------------------
PROGRAM OUTPUT
------------------------------

Output from './Build test':

t/00-compile.t ..................... ok
Log4perl: Seems like no initialization happened. Forgot to call init()?
t/05_condition_nested.t ............ ok
t/action.t ......................... ok
t/action_field.t ................... ok
t/action_mailer.t .................. ok
t/action_null.t .................... ok
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 244) line 1.
# Looks like your test exited with 2 before it could output anything.
t/action_validate.t ................ 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 13/13 subtests 
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 242) line 1.
# Looks like your test exited with 2 just after 1.
t/add_config_bug.t ................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 3/4 subtests 
t/author-critic.t .................. skipped: these tests are for testing by the author
t/author-pod-coverage.t ............ skipped: these tests are for testing by the author
t/author-pod-syntax.t .............. skipped: these tests are for testing by the author
t/base.t ........................... ok
Cannot include condition class 'TestCachedApp::Condition::EvenCounts': Can't locate TestCachedApp/Condition/EvenCounts.pm in @INC (you may need to install the TestCachedApp::Condition::EvenCounts module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 244) line 1.
# Looks like your test exited with 2 before it could output anything.
t/cached_conditions.t .............. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 12/12 subtests 
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 246) line 1.
# Looks like your test exited with 2 just after 3.
t/condition.t ...................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 8/11 subtests 
t/condition_evaluate.t ............. ok
t/config.t ......................... ok
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 244) line 1.
# Looks like your test exited with 2 before it could output anything.
t/context-persists.t ............... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 14/14 subtests 
t/context.t ........................ ok
t/exception.t ...................... ok

#   Failed test at t/factory.t line 30.
# died: Workflow::Exception::Configuration (Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 249) line 1.
# )

#   Failed test at t/factory.t line 35.
# died: Workflow::Exception::Configuration (Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 250) line 1.
# )
# Looks like you failed 2 tests of 7.
t/factory.t ........................ 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/7 subtests 

#   Failed test 'Attempt to create known workflow via callback succeeds'
#   at t/factory_callback_config.t line 39.
# died: Workflow::Exception::Configuration (Cannot include condition class 'TestApp::Condition::AlwaysTrue': Can't locate TestApp/Condition/AlwaysTrue.pm in @INC (you may need to install the TestApp::Condition::AlwaysTrue module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 251) line 1.
# )
# Looks like you failed 1 test of 7.
t/factory_callback_config.t ........ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/7 subtests 

#   Failed test 'require FactorySubclass;'
#   at t/factory_subclass.t line 10.
#     Tried to require 'FactorySubclass'.
#     Error:  Can't locate FactorySubclass.pm in @INC (you may need to install the FactorySubclass module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 235) line 2.
Can't locate object method "instance" via package "FactorySubclass" at t/factory_subclass.t line 11.
# Looks like your test exited with 255 just after 1.
t/factory_subclass.t ............... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 5/5 subtests 
t/history.t ........................ ok
t/persister.t ...................... ok
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 251) line 1.
# Looks like your test exited with 2 just after 1.
t/persister_dbi.t .................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 42/43 subtests 
t/persister_dbi_extra_data.t ....... ok
Can't locate TestDBUtil.pm in @INC (you may need to install the TestDBUtil module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at t/persister_dbi_sqlite.t line 10.
BEGIN failed--compilation aborted at t/persister_dbi_sqlite.t line 10.
t/persister_dbi_sqlite.t ........... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 246) line 1.
# Looks like your test exited with 2 just after 1.
t/persister_file.t ................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 18/19 subtests 
t/persister_random_id.t ............ ok
t/persister_spops.t ................ skipped: SPOPS not installed
t/persister_uuid.t ................. ok
t/release-cpan-changes.t ........... skipped: these tests are for release candidate testing
t/release-kwalitee.t ............... skipped: these tests are for release candidate testing
t/release-meta-json.t .............. skipped: these tests are for release candidate testing
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 244) line 1.
# Looks like your test exited with 2 just after 1.
t/state.t .......................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 20/21 subtests 
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 243) line 1.
# Looks like your test exited with 2 just after 1.
t/state_perl.t ..................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 18/19 subtests 
Cannot include condition class 'TestCachedApp::Condition::EvenCounts': Can't locate TestCachedApp/Condition/EvenCounts.pm in @INC (you may need to install the TestCachedApp::Condition::EvenCounts module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 243) line 1.
# Looks like your test exited with 2 before it could output anything.
t/uncached_conditions.t ............ 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 3/3 subtests 
t/validator.t ...................... ok
t/validator_has_required_field.t ... ok
t/validator_in_enumerated_type.t ... ok
Unable to assert DateTime or similar object at t/validator_matches_date_format.t line 25.
t/validator_matches_date_format.t .. ok
Cannot include condition class 'TestApp::Condition::HasUser': Can't locate TestApp/Condition/HasUser.pm in @INC (you may need to install the TestApp::Condition::HasUser module) (@INC contains: t /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/lib /home/david/cpantesting/perl-5.30.2/.cpan/build/Workflow-1.49-0/blib/arch /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2 /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux /home/david/cpantesting/perl-5.30.2/lib/5.30.2) at (eval 247) line 1.
# Looks like your test exited with 2 just after 1.
t/workflow.t ....................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 36/37 subtests 

Test Summary Report
-------------------
t/action_validate.t              (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 13 tests but ran 0.
t/add_config_bug.t               (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 4 tests but ran 1.
t/cached_conditions.t            (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 12 tests but ran 0.
t/condition.t                    (Wstat: 512 Tests: 3 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 11 tests but ran 3.
t/context-persists.t             (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 14 tests but ran 0.
t/factory.t                      (Wstat: 512 Tests: 7 Failed: 2)
  Failed tests:  6-7
  Non-zero exit status: 2
t/factory_callback_config.t      (Wstat: 256 Tests: 7 Failed: 1)
  Failed test:  7
  Non-zero exit status: 1
t/factory_subclass.t             (Wstat: 65280 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 5 tests but ran 1.
t/persister_dbi.t                (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 43 tests but ran 1.
t/persister_dbi_sqlite.t         (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/persister_file.t               (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 19 tests but ran 1.
t/state.t                        (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 21 tests but ran 1.
t/state_perl.t                   (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 19 tests but ran 1.
t/uncached_conditions.t          (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 3 tests but ran 0.
t/workflow.t                     (Wstat: 512 Tests: 1 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 37 tests but ran 1.
Files=42, Tests=257, 45 wallclock secs ( 0.29 usr  0.14 sys + 40.25 cusr  4.32 csys = 45.00 CPU)
Result: FAIL
Failed 15/42 test programs. 4/257 subtests failed.

------------------------------
PREREQUISITES
------------------------------

Prerequisite modules loaded:

requires:

    Module                     Need  Have    
    -------------------------- ----- --------
    Carp                       0     1.50    
    Class::Accessor            0.18  0.51    
    Class::Factory             1.00  1.06    
    Class::Observable          1.04  1.04    
    Data::Dumper               0     2.174   
    Data::UUID                 0     1.226   
    DateTime                   0.15  1.54    
    DateTime::Format::Strptime 1.00  1.78    
    DBI                        0     1.643   
    Exception::Class           1.10  1.44    
    File::Slurp                0     9999.32 
    Log::Dispatch              2.00  2.70    
    Log::Log4perl              0.34  1.53    
    perl                       5.006 5.030002
    Safe                       0     2.40    
    XML::Simple                2.00  2.25    

build_requires:

    Module                     Need  Have    
    -------------------------- ----- --------
    DBD::Mock                  0.10  1.58    
    File::Spec                 0     3.78    
    IO::Handle                 0     1.40    
    IPC::Open3                 0     1.20    
    List::MoreUtils            0     0.430   
    Module::Build              0.30  0.4231  
    Pod::Coverage::TrustPod    0     0.100005
    Test::Exception            0     0.43    
    Test::Kwalitee             1.21  1.28    
    Test::More                 0.88  1.302183
    Test::Pod                  1.41  1.52    
    Test::Pod::Coverage        1.08  1.10    

configure_requires:

    Module                     Need  Have    
    -------------------------- ----- --------
    ExtUtils::MakeMaker        0     7.34    
    Module::Build              0.30  0.4231  


------------------------------
ENVIRONMENT AND OTHER CONTEXT
------------------------------

Environment variables:

    AUTOMATED_TESTING = 1
    LANG = en_GB.UTF-8
    LANGUAGE = en_GB:en
    LC_TERMINAL = iTerm2
    LC_TERMINAL_VERSION = 3.4.3
    NONINTERACTIVE_TESTING = 1
    PATH = /home/david/perl5/perlbrew/bin:/home/david/perl5/perlbrew/perls/perl-5.30.2/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
    PERL = 5.30.2
    PERL5LIB = 
    PERL5OPT = 
    PERL5_CPANPLUS_IS_RUNNING = 29870
    PERL5_CPAN_IS_RUNNING = 29870
    PERLBREW_HOME = /home/david/.perlbrew
    PERLBREW_MANPATH = /home/david/perl5/perlbrew/perls/perl-5.30.2/man
    PERLBREW_PATH = /home/david/perl5/perlbrew/bin:/home/david/perl5/perlbrew/perls/perl-5.30.2/bin
    PERLBREW_PERL = perl-5.30.2
    PERLBREW_ROOT = /home/david/perl5/perlbrew
    PERLBREW_SHELLRC_VERSION = 0.89
    PERLBREW_VERSION = 0.89
    PERLVER = 5.30.2
    PERL_INLINE_DIRECTORY = /home/david/.Inline-5.30.2
    PERL_MM_USE_DEFAULT = 1
    PERL_USE_UNSAFE_INC = 0
    SHELL = /bin/bash
    TERM = screen

Perl special variables (and OS-specific diagnostics, for MSWin32):

    $^X = /home/david/cpantesting/perl-5.30.2/bin/perl
    $UID/$EUID = 1000 / 1000
    $GID = 1000 24 25 27 29 30 44 46 109 1000
    $EGID = 1000 24 25 27 29 30 44 46 109 1000

Perl module toolchain versions installed:

    Module              Have    
    ------------------- --------
    CPAN                2.22    
    CPAN::Meta          2.150010
    Cwd                 3.78    
    ExtUtils::CBuilder  0.280231
    ExtUtils::Command   7.34    
    ExtUtils::Install   2.14    
    ExtUtils::MakeMaker 7.34    
    ExtUtils::Manifest  1.72    
    ExtUtils::ParseXS   3.40    
    File::Spec          3.78    
    JSON                4.02    
    JSON::PP            4.02    
    Module::Build       0.4231  
    Module::Signature   n/a     
    Parse::CPAN::Meta   2.150010
    Test::Harness       3.42    
    Test::More          1.302183
    YAML                1.30    
    YAML::Syck          n/a     
    version             0.9924  


--

Summary of my perl5 (revision 5 version 30 subversion 2) configuration:
   
  Platform:
    osname=linux
    osvers=4.9.0-8-amd64
    archname=x86_64-linux
    uname='linux pigsty.barnyard.co.uk 4.9.0-8-amd64 #1 smp debian 4.9.144-3 (2019-02-02) x86_64 gnulinux '
    config_args='-de -Dprefix=/home/david/cpantesting/perl-5.30.2'
    hint=recommended
    useposix=true
    d_sigaction=define
    useithreads=undef
    usemultiplicity=undef
    use64bitint=define
    use64bitall=define
    uselongdouble=undef
    usemymalloc=n
    default_inc_excludes_dot=define
    bincompat5005=undef
  Compiler:
    cc='cc'
    ccflags ='-fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2'
    optimize='-O2'
    cppflags='-fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include'
    ccversion=''
    gccversion='6.3.0 20170516'
    gccosandvers=''
    intsize=4
    longsize=8
    ptrsize=8
    doublesize=8
    byteorder=12345678
    doublekind=3
    d_longlong=define
    longlongsize=8
    d_longdbl=define
    longdblsize=16
    longdblkind=3
    ivtype='long'
    ivsize=8
    nvtype='double'
    nvsize=8
    Off_t='off_t'
    lseeksize=8
    alignbytes=8
    prototype=define
  Linker and Libraries:
    ld='cc'
    ldflags =' -fstack-protector-strong -L/usr/local/lib'
    libpth=/usr/local/lib /usr/lib/gcc/x86_64-linux-gnu/6/include-fixed /usr/include/x86_64-linux-gnu /usr/lib /lib/x86_64-linux-gnu /lib/../lib /usr/lib/x86_64-linux-gnu /usr/lib/../lib /lib
    libs=-lpthread -lnsl -ldb -ldl -lm -lcrypt -lutil -lc
    perllibs=-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc
    libc=libc-2.24.so
    so=so
    useshrplib=false
    libperl=libperl.a
    gnulibc_version='2.24'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs
    dlext=so
    d_dlsymun=undef
    ccdlflags='-Wl,-E'
    cccdlflags='-fPIC'
    lddlflags='-shared -O2 -L/usr/local/lib -fstack-protector-strong'


Characteristics of this binary (from libperl): 
  Compile-time options:
    HAS_TIMES
    PERLIO_LAYERS
    PERL_COPY_ON_WRITE
    PERL_DONT_CREATE_GVSV
    PERL_MALLOC_WRAP
    PERL_OP_PARENT
    PERL_PRESERVE_IVUV
    USE_64_BIT_ALL
    USE_64_BIT_INT
    USE_LARGE_FILES
    USE_LOCALE
    USE_LOCALE_COLLATE
    USE_LOCALE_CTYPE
    USE_LOCALE_NUMERIC
    USE_LOCALE_TIME
    USE_PERLIO
    USE_PERL_ATOF
  Built under linux
  Compiled at Mar 16 2020 13:57:22
  %ENV:
    PERL="5.30.2"
    PERL5LIB=""
    PERL5OPT=""
    PERL5_CPANPLUS_IS_RUNNING="29870"
    PERL5_CPAN_IS_RUNNING="29870"
    PERLBREW_HOME="/home/david/.perlbrew"
    PERLBREW_MANPATH="/home/david/perl5/perlbrew/perls/perl-5.30.2/man"
    PERLBREW_PATH="/home/david/perl5/perlbrew/bin:/home/david/perl5/perlbrew/perls/perl-5.30.2/bin"
    PERLBREW_PERL="perl-5.30.2"
    PERLBREW_ROOT="/home/david/perl5/perlbrew"
    PERLBREW_SHELLRC_VERSION="0.89"
    PERLBREW_VERSION="0.89"
    PERLVER="5.30.2"
    PERL_INLINE_DIRECTORY="/home/david/.Inline-5.30.2"
    PERL_MM_USE_DEFAULT="1"
    PERL_USE_UNSAFE_INC="0"
  @INC:
    /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2/x86_64-linux
    /home/david/cpantesting/perl-5.30.2/lib/site_perl/5.30.2
    /home/david/cpantesting/perl-5.30.2/lib/5.30.2/x86_64-linux
    /home/david/cpantesting/perl-5.30.2/lib/5.30.2
```

**Comment by jonasbn** [comment]
Overview from email report:

```
Workflow-1.49:
- amd64-freebsd / 5.32.0:
 - FAIL http://www.cpantesters.org/cpan/report/eca24eb6-5592-11eb-a6fa-324b7d482270 ("David Cantrell (DCANTRELL)" <*redacted*>)

- i686-linux / 5.32.0:
 - FAIL http://www.cpantesters.org/cpan/report/37ad36d0-55a9-11eb-9e8a-2852a29fb404 ("David Cantrell (DCANTRELL)" <*redacted*>)

- x86_64-linux / 5.30.2:
 - FAIL http://www.cpantesters.org/cpan/report/0448edfe-560b-11eb-846b-3e0ef3237c92 ("David Cantrell (DCANTRELL)" <*redacted*>)

- x86_64-linux / 5.32.0:
 - FAIL http://www.cpantesters.org/cpan/report/5dc787ee-55e0-11eb-8b47-51d1f2237c92 ("David Cantrell (DCANTRELL)" <*redacted*>)

- x86_64-linux-quadmath / 5.30.2:
 - FAIL http://www.cpantesters.org/cpan/report/cbc5a57c-55dc-11eb-8503-b3baf2237c92 ("David Cantrell (DCANTRELL)" <*redacted*>)
```

**Comment by jonasbn** [comment]
In branch52 all tests have had `lib` adjusted so tests pass with: `PERL_USE_UNSAFE_INC` set to `0` (disabled).

Unsafe `INC` is default inclusion of `.` in include path. This was released with Perl 5.26, see [this blog post](https://www.effectiveperlprogramming.com/2016/06/perl-v5-26-new-features/)
